__tests__/components/Button.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Button } from '../../src/components';
+
+describe('Button', () => {
+  const onPress = { navigate: jest.fn() };
+
+  it('renders a default button', () => {
+    const tree = renderer.create(<Button title={'title'} onPress={onPress} />);
```

Comment:
for string props, you do not need javascript braces, so best practice would be `title="title"` here

Reply:
done in 9d3b4be